resolver/resolver.cc

```diff
@@ -4201,7 +4201,7 @@ vector<ast::ParsedFile> resolveSigs(core::GlobalState &gs, vector<ast::ParsedFil
     return trees;
 }
 void sanityCheck(const core::GlobalState &gs, vector<ast::ParsedFile> &trees) {
-    if (debug_mode) {
+    if (debug_mode && !skip_slow_enforce) {
```

Comment:
Why is this duplicating the `!::sorbet::skip_slow_enforce && debug_mode` check rather than just calling the macro?